pkg/rbac/userData.go

```diff
@@ -175,6 +175,8 @@ func (user *userData) getNamespacedResources(cache *Cache, ctx context.Context,
 		result, err := impersClientset.SelfSubjectRulesReviews().Create(ctx, &rulesCheck, metav1.CreateOptions{})
 		if err != nil {
 			klog.Error("Error creating SelfSubjectRulesReviews for namespace", err, ns)
+			user.nsrErr = err
+			return user, user.nsrErr
```

Comment:
Do we need to return the non-error namespaces here?

Reply:
we complete processing and send the namespaces that passed successfully .